setup.py

```diff
@@ -0,0 +1,36 @@
+"""Setup configuration for investigator."""
+
+import os
+from pathlib import Path
+from setuptools import setup
+
+
+def get_version():
+    """Get current version of adviser module."""
+    with open(os.path.join("thoth", "investigator", "__init__.py")) as f:
+        content = f.readlines()
+
+    for line in content:
+        if line.startswith("__version__ ="):
+            # dirty, remove trailing and leading chars
+            return line.split(" = ")[1][1:-2]
+    raise ValueError("No version identifier found")
+
+
+VERSION = get_version()
+setup(
+    name="thoth-investigator",
+    version=VERSION,
+    description="Thoth component used for manipulating with messages and workloads.",
```

Comment:
from the readme: "Thoth's investigator is a Kafka based component that consumes all messages produced by Thoth components." ?!

Reply:
Adjusted in 45b566f, thanks!